templates/partials/preview-question.html

```diff
@@ -0,0 +1,81 @@
+{% from 'components/panel/_macro.njk' import onsPanel %}
+
+{% set title = interviewer_note(question.title) if block.interviewer_only else question.title %}
+{% set answers = question.answers %}
+
+<div class="ons-summary__item">
+
+  <h3 class="ons-u-mt-m ons-u-fs-r--b">{{ title }}</h3>
+  
+  {% set answers_length = answers|length %}
+  
+  {% if question.descriptions %}
```

Comment:
As discussed, any constructs that are shared between this and question pages, should use a shared template/macro otherwise we will have to retest all these implementations. It also means the changes are handled in a single point making it easier to extend.

Reply:
So I spent some time on it yesterday and guidances are easy to reuse both for question and answer but instruction, description and answers are all passed into `onsQuestion` element which has a myriad of other settings and params, for example details like question type). I am not sure if it's worth adjusting `onsQuestion` for our use. For now I kept instructions, descriptions and answers as simple paragraphs,

Reply:
Only question instruction and description are passed to OnsQuestion, for those, you will need to copy the logic in the njk templates as it doesn't wrap it in `p` tags. We have to be consistent for accessibility. We will have to get this reviewed by DAC as the custom stuff most likely won't pass a DAC test.

For the rest such as guidance etc, they should be done using a shared template.

From what I can see:
- question instruction/description - copy code from templates njk not just p tags.
- question and answer guidance - shared template (already exists)
- answer label/description - can be custom maybe p tags for now with the caveat that this may fail DAC.